ansible-ipi-install/roles/installer/tasks/24_rhcos_image_cache.yml

```diff
@@ -65,7 +65,7 @@
     image: registry.centos.org/centos/httpd-24-centos7:latest
     state: started
     publish:
-      - "8080:8080"
+      - "{{ webserver_caching_port_container }}:{{ webserver_caching_port }}"
```

Comment:
I think this might be backwards.  The first port is the host port, while the second in the internal port on the container.

Reply:
@abays @jobbler - easy to confirm? I honestly copied the order from the 15_disconnected_create which will need a patch if true.

Reply:
the first one is the exposed port on the server. thats the firewall port you will open. the second is the containers port.
so 80:8080 would expose port 80 on the server.

Reply:
fixed.